README.md

```diff
@@ -1,18 +1,180 @@
-# Workleap.Eventgrid.Emulator
+# Workleap Azure Event Grid Emulator
```

Comment:
Is it normal there are so many files modified when the story was to write the readme?